backend/app/scripts/init_db.py

```
@@ -0,0 +1,25 @@
+import logging
+from falkordb import FalkorDB
+from app.core.config import settings
+from app.db.schema import init_schema
+
+# Configure basic logging for the script
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+def main():
+    logger.info(f"Connecting to FalkorDB at {settings.falkordb_url}")
```

Ruff (0.15.11)

[warning] Logging statement uses f-string

(G004)